### PR TITLE
Move UC to done

### PR DESCRIPTION
**Pull request by ThorbenStaufer**
## Basics

- [ ] I added a line to [/doc/CHANGELOG.md](/doc/CHANGELOG.md)
- [ ] The PR is rebased with current master.
- [ ] Details of what you changed are in commit messages.
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.

## Checklist

- [ ] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed the introduction tour
- [ ] I wrote migrations in a way that they are compatible with already present data
- [ ] I fixed all affected decisions
- [ ] I added unit tests for my code
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [ ] I created left-over issues for things that are still to be done
- [ ] Code is conforming to [our Architecture](/doc/architecture)
- [ ] Code is conforming to [our Guidelines](/doc/guidelines)
      (exceptions are documented)
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Examples are well chosen and understandable

### CODE REVIEW

**Review by markus2330 — CHANGES_REQUESTED** [review]
- [ ] Please write about current status (with links where code is etc.) of gain_blossoms but leave it in assigned
- [ ] write a new use case (directly in done) that describes the tour, also with links to code etc.

File: doc/usecases/done/guided_tour.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- **Error scenario:**
+  A network problem prevents the frontend from checking if the guided tour was already completed and will show an error message.
+- **Postcondition:**
+  The guided tour is completed or canceled and won't be shown again next time.
+- **Non-functional Constraints:**
+
+## Development Progress
+
+- Tour steps and display options are defined in their own [typescript file](https://github.com/ElektraInitiative/PermaplanT/blob/e4931dc6b4e1bbfaa48a6094a7c289f3cd2de57c/frontend/src/features/map_planning/utils/EditorTour.ts).
+- CSS classes for the styling of the tour were added as a seperate [CSS file](https://github.com/ElektraInitiative/PermaplanT/blob/e4931dc6b4e1bbfaa48a6094a7c289f3cd2de57c/frontend/src/styles/guidedTour.css).
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:35]: "doc/usecases/done/guided_tour.md:35: seperate ==> separate" see CI

```suggestion
- CSS classes for the styling of the tour were added as a separate [CSS file](https://github.com/ElektraInitiative/PermaplanT/blob/e4931dc6b4e1bbfaa48a6094a7c289f3cd2de57c/frontend/src/styles/guidedTour.css).
```

File: doc/usecases/done/guided_tour.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user is presented with an introduction tour upon first encountering the map editor.
+- **Status:** Done
+- **Assignee:** Thorben
+
+## Scenarios
+
+- **Precondition:**
+  - The user is registered and logged in.
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:15]:
```suggestion
  - The user is member and logged in.
```

File: doc/usecases/done/guided_tour.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# Use Case: Guided Tour
+
+## Summary
+
+- **Scope:** Gamification
+- **Level:** User Goal
+- **Actors:** App User
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:7]: Keep consistent to as used below.

```suggestion
- **Actors:** User
```

File: doc/usecases/done/guided_tour.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+- **Status:** Done
+- **Assignee:** Thorben
+
+## Scenarios
+
+- **Precondition:**
+  - The user is registered and logged in.
+  - The user has created a map.
+  - The user has opened the map editor.
+  - The user has previously not completed the guided tour.
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:18]:
```suggestion
  - The user has not previously completed the guided tour.
```

File: doc/usecases/done/guided_tour.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+- **Main success scenario:**
+  The user is shown a guided tour, that explains the features of the map editor based on a small scenario.
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:20]:
```suggestion
  The user is shown a guided tour, which explains the features of the map editor based on a small scenario.
```

File: doc/usecases/done/guided_tour.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  The user will have to place and remove plants, use the timeline, see plant relations and use the undo feature in this scenario.
+  After completing the guided tour, the user is awarded a [Blossom](assigned/gain_blossoms.md).
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:22]:
```suggestion
  After completing the guided tour, the user is awarded a [Blossom](../assigned/gain_blossoms.md).
```

File: doc/usecases/done/guided_tour.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+- **Alternative scenario:**
+  The user does not want to do the guided tour and can cancel it during any step.
+  Upon cancel, the user can decide to complete it later and see it on their next visit to the map editor or cancel it indefinitely.
```
Comment (markus2330) [inline doc/usecases/done/guided_tour.md:25]:
```suggestion
  Upon cancel, the user decides to see it on their next visit or to cancel it indefinitely.
```

**Review by markus2330 — CHANGES_REQUESTED** [review]
Great job, beautiful summary of where is what. Please also add changelog. Please also add left-over issues (translation of guided tour).

**Comment by ThorbenStaufer** [comment]
> translation of guided tour

Tour translation already has an open issue with #710.

**Review by markus2330 — APPROVED** [review]
> Tour translation already has an open issue with https://github.com/ElektraInitiative/PermaplanT/issues/710.

Yes, this should be linked in the UC (see also other done UC), as it is not possible in GitHub to filter for issues related to a UC.

**Comment by markus2330** [comment]
jenkins build please

**Pull request merged**